tool/merkle_calculator/merkle_tree_calculator.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"time"
+
+	"github.com/iotaledger/iota.go/consts"
+	"github.com/iotaledger/iota.go/guards"
+	"github.com/iotaledger/iota.go/merkle"
+	"github.com/iotaledger/iota.go/tool"
+	"github.com/iotaledger/iota.go/trinary"
+)
+
+func main() {
+
+	var (
+		depth         = flag.Int("depth", 0, "Depth of the Merkle tree to create")
+		securityLevel = flag.Int("securityLevel", 0, "Security level of the private key used")
+		seed          = flag.String("seed", "", "Seed for leaves derivation")
+		outputPath    = flag.String("output", "", "Output file path")
+	)
+
+	flag.Parse()
+
+	if *depth < 1 {
+		log.Panicf("'depth' cannot be lower than 1")
+		return
+	}
+
+	if *securityLevel < 1 || *securityLevel > 3 {
+		log.Panicf("'securityLevel' must be 1, 2 or 3")
+		return
+	}
+
+	if !guards.IsTrytesOfExactLength(*seed, consts.HashTrytesSize) {
+		log.Panicf("'seed' must be a string of 81 trytes")
+		return
+	}
+
+	if *outputPath == "" {
+		log.Panicf("'output' is required")
+		return
+	}
+
+	log.Printf("calculating %d addresses...\n", 1<<uint(*depth))
+
+	ts := time.Now()
+
+	mt := merkle.CreateMerkleTree(trinary.Hash(*seed), *securityLevel, *depth)
```

Comment:
```suggestion
	mt := merkle.CreateMerkleTree(*seed, *securityLevel, *depth)
```

trinary.Hash is already a string